terminal_check_bsd.go

```diff
@@ -1,4 +1,4 @@
-// +build darwin dragonfly freebsd netbsd openbsd
+// +build darwin dragonfly freebsd netbsd openbsd hurd
```

Comment:
> `terminal_check_bsd.go`

Looks like the filename now should be changed?

Reply:
Actually, no: the hurd libc API is largely very bsd-ish, it even defines BSD